classes/01class/exercises/c01-aws01/shahriar048/README.md

```diff
@@ -0,0 +1,330 @@
+                        "MacAddress": "06:b2:96:05:83:a2",
+                        "NetworkInterfaceId": "eni-00fb95d55552b9e0e",
+                        "OwnerId": "410368138576",
+                        "PrivateDnsName": "ip-172-31-45-101.ap-southeast-2.compute.internal",
+                        "PrivateIpAddress": "172.31.45.101",
+                        "PrivateIpAddresses": [
+                            {
+                                "Primary": true,
+                                "PrivateDnsName": "ip-172-31-45-101.ap-southeast-2.compute.internal",
+                                "PrivateIpAddress": "172.31.45.101"
+                            }
+                        ],
+                        "SourceDestCheck": true,
+                        "Status": "in-use",
+                        "SubnetId": "subnet-969e7cde",
+                        "VpcId": "vpc-49e2eb2e",
+                        "InterfaceType": "interface"
+                    }
+                ],
+                "RootDeviceName": "/dev/xvda",
+                "RootDeviceType": "ebs",
+                "SecurityGroups": [
+                    {
+                        "GroupName": "my-sg",
+                        "GroupId": "sg-0d9198b56e119769d"
+                    }
+                ],
+                "SourceDestCheck": true,
+                "StateReason": {
+                    "Code": "pending",
+                    "Message": "pending"
+                },
+                "VirtualizationType": "hvm",
+                "CpuOptions": {
+                    "CoreCount": 1,
+                    "ThreadsPerCore": 1
+                },
+                "CapacityReservationSpecification": {
+                    "CapacityReservationPreference": "open"
+                },
+                "MetadataOptions": {
+                    "State": "pending",
+                    "HttpTokens": "optional",
+                    "HttpPutResponseHopLimit": 1,
+                    "HttpEndpoint": "enabled"
+                }
+            }
+        ],
+        "OwnerId": "410368138576",
+        "ReservationId": "r-06cb45e5b16fd374d"
+    }
+```
+
+- Commands to connect to the first EC2 instance:
+
+```
+$ `ssh -i MyKeyPair.pem ec2-user@13.211.173.223`
+
+Output:
+    The authenticity of host '13.211.173.223 (13.211.173.223)' can't be established.
+    ECDSA key fingerprint is SHA256:NK7gpGoTPUEeg6rENKJWLqnP18JnvSVCpk/ZPOAUFtk.
+    Are you sure you want to continue connecting (yes/no/[fingerprint])? yes
+    Warning: Permanently added '13.211.173.223' (ECDSA) to the list of known hosts.
+
+        __|  __|_  )
+        _|  (     /   Amazon Linux AMI
+        ___|\___|___|
+
+    https://aws.amazon.com/amazon-linux-ami/2018.03-release-notes/
+    5 package(s) needed for security, out of 7 available
+    Run "sudo yum update" to apply all updates.
+```
+
+- Commands to create the second EC2 instance and any additional resource required:
+
+```
+$ `aws ec2 run-instances \
```

Comment:
considering a more secure security group than the one created previously can you also make a more secure SG for this private instance?
Feel free to create a different SG for this instance only